Doc Sweep AspNetDocs (#513)
dotnet/AspNetDocs: aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/deploying-web-packages.md

1. Build and package the web application project, as described in [Building and Packaging Web Application Projects](building-and-packaging-web-application-projects.md).
2. Modify the *ContactManager.Mvc.SetParameters.xml* file to contain the correct parameter values for your staging environment, as described in [Configuring Parameters for Web Package Deployment](configuring-parameters-for-web-package-deployment.md).
3. Open a Command Prompt window and browse to the location of MSDeploy.exe. This is typically at %PROGRAMFILES%\IIS\Microsoft Web Deploy V2\msdeploy.exe.

> 4. Type this command, where `$CREDENTIAL_PLACEHOLDER$` is a placeholder for the password key-value pair. Then press Enter (disregard the line breaks):

Reviewer: revert

> 4. Type this command, where `$CREDENTIAL_PLACEHOLDER$` is a placeholder for the password key-value pair. Then press Enter (disregard the line breaks):

Author: ```suggestion
4. Type this command, and then press Enter (disregard the line breaks):
```
